### ISSUE TITLE
muldiple IXXAT USB to Can V2 compact channels

### ISSUE THREAD

**Issue by gxurma** [rbei-etas/busmaster#769]
I have to log 3 engines with 3 new USB-to-CAN-V2 compact boxes.
Driver is the latest. 3.5 VCI: 3.5.2.4072

I only see the first channel somehow correctly.
CAN1:

**Comment by RBEI-Saravanan** [comment]
Unfortunately, we couldn't check this as we don't possess this device. Would you know someone who can help to solve this issue ?

**Comment by gxurma** [comment]
no. Only me. If I can get back my 3 devices, I maybe can contribute. Only thing I know about those devices is, that they are using a C_Can compatible chip, and it is using the recent VCI driver from ixxat.

**Comment by gxurma** [comment]
I got my 3 IXXAT USB 2 CAN V2 back from field. Can help you now. What exactly shall I do?
